app_server.py

```diff
@@ -270,9 +270,13 @@ def work_lookup(self, annotator, route_name='lookup', **process_urn_kwargs):
         urns = flask.request.args.getlist('urn')
 
         this_url = cdn_url_for(route_name, _external=True, urn=urns)
-        self.process_urns(urns)
+        response = self.process_urns(urns)
         self.post_lookup_hook()
 
+        if response:
+            # In a subclass, self.process_urns may return a ProblemDetail
+            return response
```

Comment:
Can you add a docstring to process_urns saying that it can `:return:` a ProblemDetail or a Response if it wants to override the default behavior?